### DISCUSSION TITLE
Module parse failed: Unexpected token

### DISCUSSION THREAD

**Discussion by tonkunz** [dm4t2/vue-currency-input#243] [Q&A] [answered]
Hi Everyone,
First of all, thanks for creating this component!

I try to install the latest version in my vue project with TS ("vue": "3.0.11", "typescript": "~4.1.5") and I don't know what I'm doing wrong, because the development server always give that message: 
![Captura de tela de 2021-06-14 16-03-30](https://user-images.githubusercontent.com/8299111/121945182-7c6c0180-cd21-11eb-8af7-77a11ceea5a5.png)

When I remove completely the component that use the vue-currency-input implementation, the project starts/works normally.

By this time, ty!

**Comment by dm4t2** [comment, marked as answer]
Hi, see #228. 

This issue affects only the following versions:

- 2.0.0-rc.5
- 2.0.0
- 2.0.1 

Fixed in [2.0.2](https://github.com/dm4t2/vue-currency-input/releases/tag/2.0.2) as the library is now compiled to ES2018 (instead of ES2020) to improve the compatibility.
